tests/behaviour/cube.feature

```diff
@@ -160,404 +160,475 @@ Feature: Cube!
+          "Value": 3,
+          "Measure": "length",
+          "Unit": "cm"
+        },
+        {
+          "Dimension": "e",
+          "Value": 4,
+          "Measure": "length",
+          "Unit": "cm"
+        }
+      ]
+      """
+
+  Scenario: Output a cube when an inline code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
+    Given The config json file "v1.1/cube_config_inline_code_list_with_concepts_defined_elsewhere.json" and the existing tidy data csv file "v1.1/data.csv"
+    When The cube is created
+    Then The cube Metadata should match
+      """
       {
-        "Dimension":"e",
-        "Value":4,
-        "Measure":"length",
-        "Unit":"cm"
+      "title": "Test cube config",
+      "identifier": None,
+      "summary": "Test cube config summary",
+      "description": "Test cube config description",
+      "creator_uri": "http://statistics.data.gov.uk",
+      "publisher_uri": "http://statistics.data.gov.uk",
+      "landing_page_uris": [],
+      "theme_uris": [],
+      "keywords": ["Test cube"],
+      "dataset_issued": "2022-04-08",
+      "dataset_modified": None,
+      "license_uri": "https://creativecommons.org/licenses/by/4.0/",
+      "public_contact_point_uri": None,
+      "uri_safe_identifier_override": None
       }
-    ]
-    """
-    
-    Scenario: Output a cube when an inline code list is defined using code list config schema v1.0 and when there are references to concepts defined elsewhere.
-    Given The config json file "v1.1/cube_config_inline_code_list_with_concepts_defined_elsewhere.json" and the existing tidy data csv file "v1.1/data.csv"
+      """
+    Then The cube Columns should match
+      """
+      [
+        "Dimension",
+        "Value",
+        "Measure",
+        "Unit"
+      ]
+      """
+    Then The cube data should match
+      """
+      [
+        {
+          "Dimension": "a",
+          "Value": 0,
+          "Measure": "length",
+          "Unit": "cm"
+        },
+        {
+          "Dimension": "b",
+          "Value": 1,
+          "Measure": "length",
+          "Unit": "cm"
+        },
+        {
+          "Dimension": "c",
+          "Value": 2,
+          "Measure": "length",
+          "Unit": "cm"
+        },
+        {
+          "Dimension": "d",
+          "Value": 3,
+          "Measure": "length",
+          "Unit": "cm"
+        },
+        {
+          "Dimension": "e",
+          "Value": 4,
+          "Measure": "length",
+          "Unit": "cm"
+        }
+      ]
+      """
+
+  Scenario: Successfully outputs a cube using schema v1.2
+    Given The config json file "v1.2/qudt-unit-template-testing.json" and the existing tidy data csv file "v1.2/data.csv"
+    When The cube is created
+    Then There are no errors
+
+  Scenario: Output a cube when the code list is defined using code list config schema v1.1 with concepts in which the uri_safe_identifier_override is defined.
```

Comment:
I take it all of the changes above are just whitespace alterations.

Reply:
Yes that is correct